src/utils/shortId.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { shortId } from 'utils/shortId.js'
+
+describe('shortId()', () => {
+	describe('it should reduce the string input to only the three first characters in the string ', () => {
+		it.each([
+			['Example', 'Exa'],
+			['Test', 'Tes'],
+			['robotId', 'rob'],
+			['Hello', 'Hel'],
```

Comment:
Since it is going to be used to shorten ids, at least use one example Id here.

Also in your tests, think a bit about edge cases. What happens if the id is shorter than 3 characters? Right now all you tests are testing the same property of the function: shorting a string that is longer than 3 characters to a string that is exactly 3 characters.